### ISSUE TITLE
Improve Downpage [FEATURE]

### ISSUE THREAD

**Issue by Kartikay304** [satu0king/CircuitVerse-Downpage#5]
**Is your feature request related to a problem? Please describe.**
To make the down-page more responsive.

**Describe the solution you'd like**

1. Add a countdown feature with an animated timer.
2. Add a submission form with **Notify Us** button to notify users when the website comes online.
3. Add social media icons with hover effects.

**Additional context**
Example -

![under-construction-7](https://user-images.githubusercontent.com/40604582/132105284-b68d7951-9718-43cf-a84b-f74f3b2cb285.jpg)

**Are you working on this? (Yes/No)**
Currently not, but would like to work on the feature once approved by maintainers.

**Comment by lostgirljourney** [comment]
@Kartikay304 yes, the downpage should but be updated and if it happens then I would like to contribute to it too. Need some UI working too.

**Comment by sharko7** [comment]
Can I work on this?

**Comment by manavsiddharthgupta** [comment]
/ assign me

**Comment by sambhavgupta0705** [comment]
Hey,Can I work on this issue?

**Comment by AtharvSabde** [comment]
I would like to work on this issue.  I have made many cool hovering effect animations

**Comment by yadavshubham01** [comment]
Is your feature request related to a problem? Please describe.
To make the down-page more responsive.

Describe the solution you'd like

Add a countdown feature with an animated timer.
Add a submission form with Notify Us button to notify users when the website comes online.
Add social media icons with hover effects.
Additional context
Example -
![template_04](https://github.com/CircuitVerse/CircuitVerse-Downpage/assets/126192924/5618be48-8d64-4e69-a86f-748acee91992)

**Comment by yadavshubham01** [comment]
Is your feature request related to a problem? Please describe.
To make the down-page more responsive.

Describe the solution you'd like

Add a countdown feature with an animated timer.
Add a submission form with Notify Us button to notify users when the website comes online.
Add social media icons with hover effects.
Additional context
Example -
![template_12](https://github.com/CircuitVerse/CircuitVerse-Downpage/assets/126192924/b06e6679-b4c4-4d34-89b0-d2cc909d451e)

**Comment by ghost** [comment]
is this issue is still open as i want to work on this ?

**Comment by vishu56096-ctrl** [comment]
@Kartikay304  i want to work on this issue

**Comment by vishu56096-ctrl** [comment]
@satu0king  i want to work on issue plz assign me
